lib/recipe/widgets/recipe_screen.dart

```diff
@@ -17,69 +16,27 @@ class RecipeScreen extends StatelessWidget {
     return BlocBuilder<RecipeBloc, RecipeState>(builder: (context, state) {
       return Scaffold(
           appBar: AppBar(
-            title: assignedFoodItemId == null
-                ? const Text('Add a new recipe')
-                : const Text('Confirm chosen recipe'),
+            title: const Text('Edit recipe'),
```

Comment:
Apart from changing the filename and the class to `RecipeIngredientsScreen`, there is also some discrepancy between the AppBar title of this screen and those for details and probes.

Here the bottom nav. bar says recipe ingredients and the title is Edit recipe:
<img width="591" alt="image" src="https://user-images.githubusercontent.com/45038919/153198125-9f4d4496-f053-44f1-bb4e-15a76a8d16ed.png">

While in `Probes` it is `Probe list for $recipe_name$`, even though we're still editing the recipe.
<img width="547" alt="image" src="https://user-images.githubusercontent.com/45038919/153198233-78a926b6-058d-4c39-8411-fc3fd16fe203.png">

And in `Recipe Details` it is just `Recipe Details`:
<img width="547" alt="image" src="https://user-images.githubusercontent.com/45038919/153198299-490faf3e-54be-49f4-9d3b-e9e297a810c8.png">

I think it would be nicer to have it more consistent, so maybe either changing _this_ title for `Recipe Ingredients` to `Recipe ingredients` or maybe leaving the `Edit recipe` title across all three screnes, or maybe even having the titles as: `Edit recipe probes`, `Edit recipe ingredients` and `Edit recipe details` for the three screens.